apiv2/support/enums.py

```diff
@@ -11,21 +11,21 @@
 
 
 @strawberry.enum
-class annotation_file_source_enum(enum.Enum):
+class annotation_file_source_enum(enum.StrEnum):
```

Comment:
I believe we should also handle for case where an annotation submitted by the dataset_author is the portal_standard.

Reply:
I'm trying to keep most of the schema changes (except what's specifically necessary to support the db import script) to the #277 PR - I'll add a note there!